tests/framework/i18n/FormatterDateTest.php

```diff
@@ -88,6 +88,32 @@ public function testAsDate()
         $this->assertSame($this->formatter->nullDisplay, $this->formatter->asDate(null));
     }
 
+    public function testIntlAsDateOtherCalendars()
+    {
+        // Persian calendar
+        $this->formatter->locale = 'fa_IR@calendar=persian';
+        $this->formatter->calendar = \IntlDateFormatter::TRADITIONAL;
+        $this->formatter->timeZone = 'UTC';
+
+        $value = 1451606400; // Fri, 01 Jan 2016 00:00:00 (UTC)
+        $this->assertSame('۱۳۹۴/۱۰/۱۱ ه‍.ش.', $this->formatter->asDate($value, 'short'));
```

Comment:
this fails on travis but works on my computer, seems icu version makes this untestable...